frontend/apps/app/components/Chat/MentionSuggestor/types.ts

```diff
@@ -0,0 +1,29 @@
+import type * as React from 'react'
+import type { ReactNode } from 'react'
+
+export type MentionCandidate = {
```

Comment:
[IMO]
Personally, I felt that `MentionSuggestionItem` was more intuitive and easier to understand.


```suggestion
export type MentionSuggestionItem = {
```